src/utils/runtime/src/lib.rs

```diff
@@ -119,8 +125,27 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
 
         fmt_layer.with_filter(filter)
     };
+    let mut layers = vec![fmt_layer.boxed()];
+
+    if ENABLE_QUERY_LOG_FILE {
+        let query_log_path = ".risingwave/log/query.log";
```

Comment:
I wanted to make it a configurable path, but didn't find a good place. 😇

Using env var in `main_okk` seems ok, but logger init is outside of `main_okk`. 😇